cosmic_ray/work_db.py

```diff
@@ -149,7 +149,9 @@ def pending_work(self):
         """The sequence of pending `WorkItem`s in the session."""
         table = self._work_items
         work_item = tinydb.Query()
-        pending = table.search(work_item.worker_outcome == None)
+        out = work_item.worker_outcome
```

Comment:
This whole change feels a lot more confusing than the original. Is there a reason for it?

Reply:
otherwise we'll not fit in 80 chars :sad:

Reply:
I'd much rather exceed 80 chars than use this confusing version.

Reply:
I'd really like this changed back to how it was.

Reply:
@abingham the problem is that then it will not fit within the number of columns allowed, and the linting will fail

Reply:
Then the linting is broken.

Reply:
What I mean by this is that, in this case, linting is asking us to do the wrong thing. Expressive, readable code is more important than obeying the rules in the linter.